crates/tags/src/hash.rs

```diff
@@ -33,6 +35,12 @@ impl FromStr for Hash {
     }
 }
 
+impl Serialize for Hash {
+    fn serialize<S: serde::Serializer>(&self, serializer: S) -> Result<S::Ok, S::Error> {
+        self.0.to_string().serialize(serializer)
+    }
+}
```

Comment:
You can `#[derive(Serialize)]` since `drawbridge_hash:Hash` already implements `Serialize`.

Reply:
`drawbridge_hash::Hash` doesn't have `Serialize` https://github.com/profianinc/drawbridge/blob/main/crates/hash/src/hash.rs#L124

Reply:
Fixed